src/main/java/jenkins/plugins/slack/StandardSlackService.java

```diff
@@ -88,6 +101,26 @@ public boolean publish(String message, String color) {
         return result;
     }
 
+    private String getTokenToUse() {
+        if (credentialsAuthTokenId != null && !credentialsAuthTokenId.isEmpty()) {
+            StringCredentials credentials = lookupCredentials(credentialsAuthTokenId);
+            if (credentials != null) {
+                logger.info("Using secret for Slack auth token.");
```

Comment:
Yes, I did.  There are a bunch of logging statements in the plugin, I figured this could be a useful bit of logging information.